apps/prairielearn/src/pages/editError/editError.html.ts

```diff
@@ -0,0 +1,102 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { config } from '../../lib/config.js';
+import type { JobSequenceWithFormattedOutput } from '../../lib/server-jobs.js';
+
+export function EditError({
+  resLocals,
+  jobSequence,
+  failedSync,
+}: {
+  resLocals: any;
+  jobSequence: JobSequenceWithFormattedOutput;
+  failedSync: boolean;
+}) {
+  const { __csrf_token } = resLocals;
+
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(import.meta.url, "<%- include('../partials/head'); %>", {
+          ...resLocals,
+          pageTitle: 'Edit Failure',
+        })}
+      </head>
+
+      <body>
+        ${renderEjs(import.meta.url, "<%- include('../partials/navbar'); %>", {
+          ...resLocals,
+          navPage: 'error',
+        })}
+
+        <main id="content" class="container">
+          <script>
+            $(function () {
+              const button = document.querySelector('#results-some-uuid-button');
+              button.addEventListener('click', () => {
+                button.textContent = document
+                  .querySelector('#results-some-uuid')
+                  .classList.contains('show')
+                  ? 'Show detail'
+                  : 'Hide detail';
+              });
+            });
+          </script>
+
+          <div class="card mb-4">
+            <h5 class="card-header bg-danger text-white">
+              <div class="row align-items-center justify-content-between">
+                <div class="col-auto">Edit Failure</div>
+                <div class="col-auto">
+                  <button
+                    type="button"
+                    class="btn btn-light btn-sm"
+                    data-toggle="collapse"
+                    data-target="#results-some-uuid"
+                    id="results-some-uuid-button"
+                  >
+                    Show detail
+                  </button>
+                </div>
+              </div>
+            </h5>
+            <div class="card-body">
+              <p>The file edit did not complete successfully.</p>
+              ${failedSync
+                ? html`
+                    <p>
+                      In particular, it looks like your changes were written to
+                      disk${config.fileEditorUseGit
+                        ? 'and were pushed to the remote GitHub repository'
```

Comment:
I think you're missing a space here.

```suggestion
                        ? ' and were pushed to the remote GitHub repository'
```